src/charm.py

```diff
@@ -136,6 +137,27 @@ def _configure(self, _) -> None:
             self._write_config_file(content=ue_config)
         self._configure_pebble(restart=service_restart_required)
 
+    def _on_start_simulation_action(self, event: ActionEvent) -> None:
+        """Run network traffic simulation.
+
+        The action tries to ping `8.8.8.8` using the UE interface (oaitun_ue1). Working ping
```

Comment:
We may want to parametrize the ping destination at some point, but for now that's ok

Reply:
I was thinking about this, but I don't think we'll ever need that. Unless we start testing in a super closed env without internet access.

Reply:
Exact, it would be for those types of use cases, but like I mentioned, no need to do this right now.